### ISSUE TITLE
FR: use-output-decorator

### ISSUE THREAD

**Issue by rafaelss95** [angular-eslint/angular-eslint#474]
**Description and reproduction of the issue**
Currently if we have an `EventEmitter` without `@Output()` it doesn't work as expected (sometimes we just forget to annotate the properties 😄 ).

```ts
export class AppComponent  {
  // Note that the `@Output` is missing
  readonly test = new EventEmitter<void>();
}
```

```html
<my-app (test)="test()"></my-app>
```

The rule should find `EventEmitter` occurrences without `@Output` decorator and fix it.

**Reproduction**:
https://stackblitz.com/edit/angular-ivy-xsxvax
